cli/crates/server/src/introspect_local.rs

```diff
@@ -23,9 +23,18 @@ pub async fn introspect_local() -> Result<IntrospectLocalOutput, ServerError> {
 
     let rendered_sdl = registry.export_sdl(registry.enable_federation);
 
+    let rendered_sdl = prettify(rendered_sdl);
+
     if federated_graph_config.is_some() && rendered_sdl.is_empty() {
         Ok(IntrospectLocalOutput::EmptyFederated)
     } else {
         Ok(IntrospectLocalOutput::Sdl(rendered_sdl))
     }
 }
```

Comment:
Maybe safer to not try and prettify an empty schema.  I assume it would always fail to parse.  Assume it would always fail to parse and just return the empty string again, but why even try

```suggestion
    if federated_graph_config.is_some() && rendered_sdl.is_empty() {
        Ok(IntrospectLocalOutput::EmptyFederated)
    } else {
        Ok(IntrospectLocalOutput::Sdl(prettify(rendered_sdl);))
    }
}
```

Reply:
Agreed, updated